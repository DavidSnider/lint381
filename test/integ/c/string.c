struct foo; // should be flagged
"struct foo;" // should not be flagged
struct foo; // should be flagged
