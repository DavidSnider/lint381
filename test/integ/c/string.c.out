string.c:1:8: error: Struct name 'foo' should be capitalized
struct foo; // should be flagged
       ^^^
string.c:3:8: error: Struct name 'foo' should be capitalized
struct foo; // should be flagged
       ^^^
